doc/nrf/protocols/matter/overview/group_communication.rst

```diff
@@ -0,0 +1,228 @@
+            epochStartTime1 = 2220001
+            epochKey2 = f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff
+            epochStartTime2 = 2220002
+        KeySetId 0x1a1 Epochs keys:
+            epochKey0 = a0a1a2a3a4a5a6a7a8a9aaabacadaeaf
+            epochStartTime0 = 1110000
+            epochKey1 = b0b1b2b3b4b5b6b7b8b9babbbcbdbebf
+            epochStartTime1 = 1110001
+            epochKey2 = c0c1c2c3c4c5c6c7c8c9cacbcccdcecf
+            epochStartTime2 = 1110002
+
+* To see the groups, use the following command:
+
+  .. code-block:: none
+
+     ./chip-tool groupsettings show-groups
+     ...
+       +-------------------------------------------------------------------------------------+
+       | Available Groups :                                                                  |
+       +-------------------------------------------------------------------------------------+
+       | Group Id   |  KeySet Id     |   Group Name                                          |
+       | 0x101           0x1a1            Group #1                                           |
+       | 0x102           0x1a2            Group #2                                           |
+       | 0x103           0x1a3            Group #3                                           |
+       +-------------------------------------------------------------------------------------+
+
+Complete the following steps to create a custom group based on a custom keyset and start Group Communication:
+
+1. Generate a custom keyset::
+
+   ./chip-tool groupsettings add-keysets 0xabcd 0 3330000 hex:0123456789abcdef0123456789abcdef
+   ./chip-tool groupsettings bind-keyset 0xdcba 0xabcd
+
+#. Set the keyset on the Light Bulb based on generated controller's group settings::
+
+   ./chip-tool groupkeymanagement key-set-write '{"groupKeySetID":"0xabcd","groupKeySecurityPolicy":0,"epochKey0":"0123456789abcdef0123456789abcdef","epochStartTime0":"3330000","epochKey1":"0123456789abcdef0123456789abcdee","epochStartTime1":"3330001","epochKey2":"0123456789abcdef0123456789abcded","epochStartTime2":"3330002"}' 1 0
+
+#. Set the keyset to the group map::
+
+   ./chip-tool groupkeymanagement write group-key-map '[{"groupId":"0xdcba","groupKeySetID":"0xabcd","fabricIndex":"1"}]' 1 0
+
+#. Add the group on the application endpoint (endpoint 1 includes the on/off cluster)::
+
+   ./chip-tool groups add-group 0xdcba Custom_Group 1 1
+
+#. Set the ACL to permit the Group Communication from a specific group ID::
+
+   ./chip-tool accesscontrol write acl '[{"fabricIndex":"1","privilege":"5","authMode":"2","subjects":["112233"],"targets":null},{"fabricIndex":"1","privilege":"3","authMode":"3","subjects":["0xdcba"],"targets":[{"cluster":null,"endpoint":"1","deviceType":null}]}]' 1 0
+
+#. Send a command to toggle on/off on the application endpoint (endpoint 1 includes the onoff cluster)::
+
+   ./chip-tool onoff toggle 0xffffffffffffdcba 1
+
+**LED2** on the Light Bulb should now change the state.
```

Comment:
It is LED 2 for nrf53/nrf52,  for nrf54l15 is LED1 (because numeration is different on DK)